### ISSUE TITLE
Can't re-display image

### ISSUE THREAD

**Issue by traviswimer** [traviswimer/PlugPro#10]
If you hide an inline image there is no way to re-display it. Not a very important problem, but would be nice to have.

**Comment by andrewtibbetts** [comment]
jesus. accidentally commited to master. i am still stuck down the rabbit hole...

**Comment by traviswimer** [comment]
Wait, what happened? It seems you haven't pulled any changes into your master for a while. (maybe that's what you meant) Anyway, the changes you made don't seem to be working for me. The toggle "x" button isn't clickable and the x is outside the red box.

**Comment by ghost** [comment]
I accidentally committed changes to my master and not a branch (you want all changes on branches, yes?). Finding it hard to 'undo' things in this system. #gitnoob Another thing I'm finding hard to do is know if my changes work.

**Comment by traviswimer** [comment]
Yes, it probably works best if you make the changes on separate branches. 

For undoing things, you may find this helpful: https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things

When you go to create a new branch of your master, you should do this first to make sure it is up to date: https://help.github.com/articles/syncing-a-fork/

For knowing if your changes work, are you actually loading your local changes into chrome as an extension? By that I mean doing [this](https://developer.chrome.com/extensions/faq#faq-dev-01) and then clicking "Load unpacked extension".
